frontends/web/src/routes/account/send/components/dialogs/scan-qr-dialog.tsx

```diff
@@ -0,0 +1,39 @@
+import { useTranslation } from 'react-i18next';
+import { Dialog } from '../../../../../components/dialog/dialog';
+import { Button } from '../../../../../components/forms';
+import { Spinner } from '../../../../../components/spinner/Spinner';
+import style from '../../send.module.css';
+
+type TProps = {
+    activeScanQR: boolean;
+    toggleScanQR: () => void;
+    videoLoading: boolean;
+    onLoadedVideo: () => void;
+}
+
+const DialogScanQR = ({ activeScanQR, toggleScanQR, videoLoading, onLoadedVideo }: TProps) => {
+  const { t } = useTranslation();
+  return (
+    <Dialog
+      open={activeScanQR}
+      title={t('send.scanQR')}
+      onClose={toggleScanQR}>
+      {videoLoading && <Spinner guideExists />}
+      <video
+        id="video"
+        width={400}
+        height={300 /* fix height to avoid ugly resize effect after open */}
+        className={style.qrVideo}
+        onLoadedData={onLoadedVideo} />
+      <div className={['buttons', 'flex', 'flex-row', 'flex-between'].join(' ')}>
```

Comment:
Since you are here, could you either change to a string `className="buttons flex flex-row flex-between"` or use `<DialogButtons>`?

DialogButtons will change the layout of the button a bit, since it is only 1 button it will use 100% width. I think it is ok as we use this in other places also with just 1 button.

Reply:
also looks neater w/ DialogButtons :)

Reply:
> also looks neater w/ DialogButtons :)

the code or the UI ? :)

Reply:
haha, both. IMO looks better with full width button (no unused whitespace), as the dialog isn't too wide either.

<img width="471" alt="Screenshot 2023-07-19 at 11 14 54" src="https://github.com/digitalbitbox/bitbox-wallet-app/assets/28676406/fb623966-2cc3-4d3c-a9bf-ea1390da667d">